packages/website/src/routes/layout.tsx

```diff
@@ -0,0 +1,40 @@
+import type { ReactNode } from 'react';
+
+// XXX I don't know why/how this works
+import '../style.css';
```

Comment:
This seems to work well for production (= `npm run website:prd`),
but it's very flaky in development (`npm run website:dev`).
Can anyone help investigating what's the right way in vite?